### PR TITLE
There are 4 changes in this commit:

### PR DESCRIPTION
**Pull request by plivesey**
1. Added a podspec for NeuralNet. This should make it much easier for people to install and get started.
2. Changed the sigmoid function so it doesn't break the compiler. Currently, it throws an error saying the expression is too complex.
3. Added an initializer which allows you to copy a neural network
4. When loading from JSON, assume that the numbers are Doubles. Sometimes, these numbers are not Floats so the parsing fails. I haven't really looked into this, but I think it may happen when one of the numbers is very small and so it's parsed into a double instead. This avoids any failures to load a NeuralNetwork from disk.

### CODE REVIEW

**Comment by plivesey** [comment]
I realize there's too much in this PR for one commit, but it seems like this library isn't really being maintained anymore. Happy to split this up when you get around to looking at it.

**Comment by plivesey** [comment]
Two more changes:
Added biases to the storage (thanks to #6)
Added training error to the epoch callback which is useful to see if you're overfitting the data.
